scripts/check-todo.sh

```diff
@@ -1,24 +1,23 @@
 #!/bin/bash
 
 # Continuous integration script to check that TODOs are in the correct format
-OUTPUT="$(grep -PrinH '(?<!context\.)todo(?!\(#{0,1}\d+\))' --include \*.go --exclude *site_data.go --exclude *mainnet_config.go *)";
+OUTPUT="$(grep -PrinH '(?<!context\.)todo(?!\(#{0,1}\d+\))' --include ./**/*.go --exclude ./*site_data.go --exclude ./*mainnet_config.go)";
```

Comment:
Again, will `./*site_data.go` match all those items that `*site_data.go` covers?